test/test_SW_VegProd.cc

```diff
@@ -159,4 +159,604 @@ namespace {
+            climateAverages.meanTempMon_C, climateAverages.PPTMon_cm, inputValues, shrubLimit,
+            SumGrassesFraction, C4Variables, fillEmptyWithBareGround, inNorthHem, warnExtrapolation,
+            grassOutput, RelAbundanceL0, RelAbundanceL1);
+        
+        // Loop through RelAbundanceL0 and test results
+        for(index = 0; index < 8; index++) {
+            EXPECT_NEAR(RelAbundanceL0[index], RelAbundanceL0Expected[index], tol6);
+        }
+
+        // Loop through RelAbundanceL1 and test results
+        for(index = 0; index < 5; index++) {
+            EXPECT_NEAR(RelAbundanceL1[index], RelAbundanceL1Expected[index], tol6);
+        }
+
+        EXPECT_DOUBLE_EQ(grassOutput[0], 1.);
+        EXPECT_DOUBLE_EQ(grassOutput[1], 0.);
+        EXPECT_DOUBLE_EQ(grassOutput[2], 0.);
+
+        /*  ===============================================================
+                     Test with all input values not "SW_MISSING"
+            ===============================================================  */
+
+        inputValues[succIndex] = .1098;
+        inputValues[forbIndex] = .1098;
+        inputValues[C3Index] = .1098;
+        inputValues[C4Index] = .1098;
+        inputValues[grassAnn] = .1098;
+        inputValues[shrubIndex] = .1098;
+        inputValues[treeIndex] = .1098;
+        inputValues[bareGround] = .1098;
+
+        /* Expect identical output to rSOILWAT2 (e.g., v5.3.1)
+         * NOTE: Command uses deprecated estimate_PotNatVeg_composition (rSOILWAT >= v.6.0.0)
+         ```{r}
+           rSOILWAT2::estimate_PotNatVeg_composition_old(
+             MAP_mm =  10 * clim1[["MAP_cm"]], MAT_C = clim1[["MAT_C"]],
+             mean_monthly_ppt_mm = 10 * clim1[["meanMonthlyPPTcm"]],
+             mean_monthly_Temp_C = clim1[["meanMonthlyTempC"]],
+             Succulents_Fraction = .1098, fix_succulents = TRUE,
+             C3_Fraction = .1098, fix_C3grasses = TRUE,
+             Shrubs_Fraction = .1098, fix_shrubs = TRUE,
+             Trees_Fraction = .1098, fix_trees = TRUE,
+             Annuals_Fraction = .1098, fix_annuals = TRUE,
+             C4_Fraction = .1098, fix_C4grasses = TRUE,
+             Forbs_Fraction = .1098, fix_forbs = TRUE,
+             BareGround_Fraction = .1098, fix_BareGround = TRUE
+           )
+         ```
+         */
+
+        RelAbundanceL0Expected[succIndex] = 0.1098;
+        RelAbundanceL0Expected[forbIndex] = 0.1098;
+        RelAbundanceL0Expected[C3Index] = 0.1098;
+        RelAbundanceL0Expected[C4Index] = 0.1098;
+        RelAbundanceL0Expected[grassAnn] = 0.1098;
+        RelAbundanceL0Expected[shrubIndex] = 0.1098;
+        RelAbundanceL0Expected[treeIndex] = 0.1098;
+        RelAbundanceL0Expected[bareGround] = 0.2314;
+
+        RelAbundanceL1Expected[treeIndexL1] = 0.1098;
+        RelAbundanceL1Expected[forbIndexL1] = 0.1098;
+        RelAbundanceL1Expected[shrubIndexL1] = 0.2196;

```

Comment:
besides showing the same issue as https://github.com/DrylandEcology/SOILWAT2/pull/321/files#r999543271,
please add comment to explain that L1 "forbs" represent L0 "succulents" + L0 "forbs"